src/signable/stark-signable.ts

```diff
@@ -77,8 +77,13 @@ export abstract class StarkSignable<T> {
     //
     // Compare with:
     // https://github.com/starkware-libs/starkex-resources/blob/1eb84c6a9069950026768013f748016d3bd51d54/crypto/starkware/crypto/signature/signature.py#L151
+    const isValidWithEvenY = await verify(
+      asEcKeyPairPublic(publicKey, false),
+      await this.getHashBN(),
```

Comment:
do you want to pull out `this.getHashBN()` into a variable so it can be reused here and on line 87?

Reply:
I didn't since it's cached internally but... looks like that's also a good way of helping this fit on one line